Lombiq.ContentEditors.Samples/Controllers/FrontEndDemoContentItemAsyncEditorController.cs

```diff
@@ -1,26 +1,26 @@
-﻿using Lombiq.ContentEditors.Samples.Constants;
-using Lombiq.ContentEditors.Samples.Services;
-using Lombiq.ContentEditors.ViewModels;
-using Microsoft.AspNetCore.Mvc;
-using OrchardCore.Modules;
-
-namespace Lombiq.ContentEditors.Samples.Controllers;
-
-// In case you want to create an async editor UI on a page other than the admin UI, you can create a similar controller
-// to this one.
-[Feature(FeatureIds.Samples)]
-[Route(Routes.FrontEndContentItemAsyncEditor)]
-public class FrontEndDemoContentItemAsyncEditorController : Controller
-{
-    [HttpGet("{contentItemId?}")]
-    public ActionResult Index(string contentItemId) =>
-        // You can use the existing ContentItemAsyncEditorViewModel to pass the required data.
-        View(new ContentItemAsyncEditorViewModel
-        {
-            ProviderName = nameof(SupportTicketAsyncEditorProvider),
-            ContentType = ContentTypes.SupportTicket,
-            ContentItemId = contentItemId,
-        });
-}
-
-// NEXT STATION: Views/FrontEndDemoContentItemAsyncEditor/Index.cshtml
+﻿using Lombiq.ContentEditors.Samples.Constants;
+using Lombiq.ContentEditors.Samples.Services;
+using Lombiq.ContentEditors.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using OrchardCore.Modules;
+
+namespace Lombiq.ContentEditors.Samples.Controllers;
+
+// In case you want to create an async editor UI on a page other than the admin UI, you can create a similar controller
+// to this one.
+[Feature(FeatureIds.Samples)]
+[Route(Routes.FrontEndContentItemAsyncEditor)]
+public class FrontEndDemoContentItemAsyncEditorController : Controller
```

Comment:
I get this, if I open it immediately after a fresh OSOCE setup with the Test recipe:

![image](https://github.com/Lombiq/Orchard-Content-Editors/assets/1976647/caec71e2-bc2e-496b-84fc-e5a512c4ce66)

Add a UI test for the frontend editor too and ensure that the it actually catches such errors.

Reply:
Oddly, I don't have a repro for this. Regardless, I'll add a UI test to make sure.

Reply:
This is still the case for me, it gets a 404:

![image](https://github.com/Lombiq/Orchard-Content-Editors/assets/1976647/169058e5-9891-4bd4-a4bc-7e786f7a4c93)

This is what I do:


https://github.com/Lombiq/Orchard-Content-Editors/assets/1976647/c0de2add-e9eb-4dce-8d05-37a535a346b1